JavaScript/Github_unfollowers/Github_Unfollowers.js

```diff
@@ -0,0 +1,38 @@
+// Script to track recent unfollowers on github using github api
+// Author: vismitap
+
+const prompt = require("prompt-sync")();
+const Username = prompt("What is your Github Username? : ");
+
+// require node-fetch
+const node_fetch = require("node-fetch");
+
+//require fs for memory of old and updated data
+const fs = require("fs");
+
+let url = `https://api.github.com/users/${Username}/followers`;
+
+let current = JSON.parse(fs.readFileSync("old.json"));
+console.table(current);
+
+// fetch data and store in set
+node_fetch(url)
+  .then((res) => res.json())
+  .then((out) => {
+    let i = 0;
+    let set = new Set();
+    
+    // find the followers username on github from the returned json by github API
+    for (i in out) {
+      set.add(out[i]["login"]);
+    }
+
+    if (current.length == 0) {
+      console.log("Run for the first time.");
```

Comment:
Use a more friendly terminal statement like: `Analyzed and Run for the first time`